jujugui/static/gui/src/app/components/env-switcher/list/list.js

```diff
@@ -120,29 +142,28 @@ YUI.add('env-list', function() {
 
     render: function() {
       const auth = this.props.authDetails;
-      let buttonRow;
+      let createNew;
       if (auth && auth.rootUserName) {
```

Comment:
They can't create a model if they aren't logged in so yes.

Also, it should be disabled if they don't have the correct ACL for creating models.